code/game/area/areas/prophunt.dm

```diff
@@ -0,0 +1,88 @@
+/area/prophunt
+	name = "PropHunt"
+	icon_state = "Prophunt"
+	dynamic_lighting = DYNAMIC_LIGHTING_DISABLED
+	flags_1 = 0
+	hidden = TRUE
+
+	var/obj/machinery/computer/prophunt/linked
+	var/restricted = 0 // if true, program goes on emag list
+
+/*
+	Power tracking: Use the prophunt computer's power grid
+	Asserts are to avoid the inevitable infinite loops
+*/
+
+/area/prophunt/powered(var/chan)
+	if(!requires_power)
+		return 1
+	if(always_unpowered)
+		return 0
+	if(!linked)
+		return 0
+	var/area/A = get_area(linked)
+	ASSERT(!istype(A, /area/prophunt))
+	return A.powered(chan)
+
+/area/prophunt/addStaticPower(value, powerchannel)
+	if(!linked)
+		return
+	var/area/A = get_area(linked)
+	ASSERT(!istype(A, /area/prophunt))
+	return ..()
+
+/area/prophunt/use_power(amount, chan)
+	if(!linked)
+		return 0
+	var/area/A = get_area(linked)
+	ASSERT(!istype(A, /area/prophunt))
+	return ..()
+
+
+/*
+	This is the standard prophunt.  It is intended to allow you to
+	blow off steam by doing stupid things like laying down, throwing
+	spheres at holes, or bludgeoning people.
+*/
+/area/prophunt/mg2
+	name = "\improper Recreational prophunt"
+
+/area/prophunt/mg2/offline
+	name = "PropHunt - Offline"
+
+/area/prophunt/mg2/00

```

Comment:
Set these to zero, one, two, etc so that no funny stuff happens

Reply:
When I tried doing this the code wouldn't compile properly because if you try to name the area a number value it thinks it's something else and it breaks. At least I think that's what it was from what I remember.